gradle.properties

```diff
@@ -17,7 +17,7 @@ lastCopyrightYear=2023
 
 jdkToolchainVersion=17
 runAllTestsInSeparateJVMs=false
-instrumentAllClassesInModelCheckingMode=false
+instrumentAllClasses=false
```

Comment:
Note: with this change, the Teamcity build name and configuration should be adjusted accordingly.